rpc-test-client/test/send.js

```diff
@@ -24,13 +25,42 @@ describe('Send', () => {
     it('sends 1 nano from 0x627306090abab3a6e1400e9345bc60c78a8bef57 to 0xf17f52151ebef6c7334fad080c5704d77216b732', function () {
         let account = new plasma.Account(client, web3, contract, accountAddress, accountKey);
         const amount = web3.utils.toBN(web3.utils.toWei('1', 'nano'));
-        account.Send('0xf17f52151ebef6c7334fad080c5704d77216b732', amount, (err, transaction) => {
+        const sendWrapper = function(cb) {
+            account.Send('0xf17f52151ebef6c7334fad080c5704d77216b732', amount, (err, transaction) => {
+                return cb(err, transaction);
+            });
+        };
+        async.parallel([
+            sendWrapper,
+            // sendWrapper,
+            // sendWrapper,
+            // sendWrapper,
+            // sendWrapper,
```

Comment:
Intended?

Reply:
Yes, this was a test to see concurrency when trying to spend same utxo. It uncovered some problems (which I'll fix next.)